src/types.js

```diff
@@ -108,37 +108,37 @@ function installInto(expect) {
             output.append(inspect(value.getRenderOutput()));
         }
     });
-    
-    
-    
+
+
+
     expect.addType({
        name: 'ReactTestRenderer',
        base: 'object',
        identify: function (value) {
            return value && typeof value === 'object' &&
-               value.hasOwnProperty('_component') &&
+               ({}).hasOwnProperty.call(value, '_component') &&
```

Comment:
I think given that if the object is in question is a Test Renderer, it _will_ have a prototype, so we can do as the other types do and check `typeof value.hasOwnProperty === 'function` first - that keeps it inline with the other types.